langchain4j-agentic/src/main/java/dev/langchain4j/agentic/planner/AgentInstance.java

```diff
@@ -20,4 +20,10 @@ public interface AgentInstance {
     List<AgentArgument> arguments();
 
     List<AgentInstance> subagents();
+
+    default boolean isStreaming() {
+        return false;
+    }
+
+    default void setStreaming(boolean streaming) {}
```

Comment:
This `AgentInstance` interface is exposed to users both when creating a custom `Planner` and now also as an informative model of an agent as you can see here https://github.com/langchain4j/langchain4j/blob/5774f82fca04adc77fe0b29e18f80be67fa96f7b/langchain4j-agentic/src/test/java/dev/langchain4j/agentic/WorkflowAgentsIT.java#L503

Representing the static model of an agent it shouldn't have any setter or mutable method, so please remove this.